doc/api/errors.md

```diff
@@ -1204,6 +1204,17 @@ An invalid `options.protocol` was passed.
 Both `breakEvalOnSigint` and `eval` options were set in the REPL config, which
 is not supported.
 
+<a id="ERR_INVALID_RETURN_PROPERTY"></a>
+### ERR_INVALID_RETURN_PROPERTY
```

Comment:
The name here might be better as: `ERR_INVALID_RETURN_PROPERTY_TYPE`?

Reply:
Again, the consistent name here would be `ERR_INVALID_RETURN_PROPERTY_VALUE` actually :)